docker/node.docker-compose.yml

```diff
@@ -142,34 +142,34 @@ services:
     # ports: ['${PORTPREFIX}01:80']
     command: bash -c "cd /src/tests/helpers/servers &&
       FLASK_APP=dummy_test_helper
-      flask run --host=0.0.0.0 --port=80"
+      flask run --host=0.0.0.0"
 
   message_api:
     <<: *base_api_service
     container_name: ${COMPOSE_PROJECT_NAME}_ig_message_api
     depends_on:
       - elasticmq
       - minio
-    ports: ['${PORTPREFIX}01:80']
```

Comment:
I liked the idea of `PORTPREFIX` but it's a bit crap in practice, so reverting back to explicit definition of ports for each API.